eng/scripts/Language-Settings.ps1

```diff
@@ -260,8 +260,7 @@ $PackageExclusions = @{
   '@azure/core-asynciterator-polyfill'  = 'Docs CI fails https://github.com/Azure/azure-sdk-for-js/issues/16675';
 }
 
-function Update-javascript-DocsMsPackages($DocsRepoLocation, $DocsMetadata) {
-
+function Update-javascript-DocsMsPackages($DocsRepoLocation, $DocsMetadata, $PackageSourceOverride, $DocValidationImageId=$ImageId) {
```

Comment:
If PackageSoureOverride isn't used then you don't need to define it here.

Also do we really want to default to $ImageId?

Reply:
Right, no need to add $PackageSourceOverride. I am using $ImageI for testing, will remove.